homework_1/task1/fizzbuzz/fizzbuzz_test.go

```diff
@@ -0,0 +1,45 @@
+package fizzbuzz_test
+
+import (
+	"testing"
+
+	"code-cadets-2021/homework1/task1/fizzbuzz"
+)
+
+func areSlicesEqual(first []string, second []string) bool {
+	if len(first) != len(second) {
+		return false
+	}
+
+	for idx, x := range first {
+		if x != second[idx] {
+			return false
+		}
+	}
+
+	return true
+}
```

Comment:
Even though this function is entirely correct, you don't need it for testing purposes - you can validate two slices are equal by calling `So(actual, ShouldResemble, expected)` from `goconvey` library - e.g. https://github.com/superbet-group/code-cadets-2021/blob/main/lecture_1/07_testing/filter/divisorFilterConvey_test.go

Reply:
Yea, we did two versions, for some reason i choose this one 😄